src/main/java/sg/ncl/testbed_interface/Dataset.java

```diff
@@ -76,6 +76,10 @@ public boolean isAccessible(String userId) {
         return isOpen() || isContributor(userId) || isApprovedUser(userId);
     }
 
+    public boolean isDownloadable(String userId) {
+	    return !(accessibility == DataAccessibility.QUARANTINED) && isAccessible(userId);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use the opposite operator ("!=") instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1940)